src/snowflake/cli/plugins/workspace/commands.py

```diff
@@ -33,3 +36,24 @@ def validate(
     """Validates the project definition file."""
     # If we get to this point, @with_project_definition() has already validated the PDF schema
     return MessageResult("Project definition is valid.")
+
+
+@ws.command(requires_connection=True)
+@with_project_definition()
+def bundle(
+    key: str = typer.Option(
+        help=f"""The key of the entity you want to bundle.""",
+    ),
+    **options,
```

Comment:
I know it's just temporary, but let's workshop what this "key" is going to be called before the we go too much further. Not sold on 'key' just yet.

Reply:
@sfc-gh-cgorrie Let's discuss the terminology here

Reply:
Considering it's temporary, I'd lean towards `--entity <id>` or `--entity-id <id>`.

Reply:
Changed to `--entity-id`